pkg/prowgen/prowgen_tests.go

```diff
@@ -24,6 +24,10 @@ const (
 	cronTemplate         = "%d %d * * 2,6"
 	seed                 = 12345
 	devclusterBaseDomain = "serverless.devcluster.openshift.com"
+	// Holds version of the existing cluster pool dedicated to OpenShift Serverless in CI.
+	// See https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#existing-cluster-pools
+	clusterPoolVersion       = "4.15"
+	serverlessClusterProfile = "aws-serverless"
```

Comment:
What does `aws-serverless` refer to?

Reply:
It's the name of the profile that we've been using for starting new clusters from scratch for pre-released versions of OCP (so far it was 4.15, now it will be 4.16).
It's configured [here](https://github.com/openshift/ci-tools/pull/3978/files#diff-dfee74d02fd350fb1ee1ffbbbc32acf4cb075c9ae42b6e31c8c7acefe7fed623R1237)
In the current PR, I'm only creating a new constant for this. Previously, it was used directly in code [here](https://github.com/openshift-knative/hack/blob/main/pkg/prowgen/prowgen_tests.go#L68)

Reply:
Can we maybe link that as well (see other comment below).